lib/event_listeners/addFood.js

```diff
@@ -0,0 +1,15 @@
+const $ = require('jquery')
+
+$(document).ready(function(){
+  $("#food_form").submit(function(event) {
+    var name = $("input[name=name]").val()
+    var calories = $("input[name=calories]").val()
+    if ((name.length === 0) || (calories.length === 0)) {
+      alert("To add an item, please fill out both fields.")
+    } else {
+      $("tbody").prepend("<tr><td>" + name + "</td><td>" + calories + "</td></tr>")
+      $('input').val('').removeAttr('checked').removeAttr('selected');
```

Comment:
Bam! getting a hidden gem for a card in getting the remove attr out of the way.  Balla